components/category-header.tsx

```diff
@@ -1,18 +1,32 @@
 import { useRouter } from "next/router";
+import { useState } from "react";
+import { motion } from "framer-motion";
 import Link from "next/link";
 import "./category-header.scss";
 
+const variants = {
+  initial: {},
+  whileHover: {
+    scale: 1.02
+  }
+};
+
 export const CategoryHeader = () => {
   const router = useRouter();
+  const [category, setCategory] = useState(router.query.category);
```

Comment:
@selbekk Grunnen til animasjonen hakket var fordi kategorien kom fra queryen i url-en og den ble borte med en gang man linker tilbake til forsiden. Derfor ble kategori-headeren tom mens animasjonen fortsatt pågikk. Lagrer derfor kategorien i state-en, slik at den er konstant etter at man endrer url-en. Usikker på om det er overkill, ettersom jeg aldri ønsker å endre state? Kunne jeg bare lagret det i en variabel ved første render? Måtte vel ha brukt useEffect istedet, isåfall synes jeg ikke det er mer lettvekt enn useState. Og det er jo på en måte state.